estuary_updater/handlers/errata.py

```diff
@@ -161,25 +161,32 @@ def builds_added_handler(self, msg):
 
         :param dict msg: a message to be processed
         """
-        advisory = Advisory.get_or_create({
-            'id_': msg['body']['headers']['errata_id']
-        })[0]
-        nvr = msg['body']['headers']['brew_build']
-        koji_build = self.get_or_create_build(nvr)
+        embargoed = msg['body']['headers']['product'] == 'REDACTED'
```

Comment:
Could you change this to check the following?
```python
msg['body']['headers']['brew_build'] == 'REDACTED'
```
Since that's the only value we care about that's REDACTED, it makes sense to check for that specific value for readability and if in the future they decided to not redact the product (unlikely).